pkg/module/filter.go

```diff
@@ -63,17 +63,15 @@ func (f *Filter) AddRule(path string, rule FilterRule) {
 	latest.next[last] = rn
 }
 
-// ShouldProcess evaluates path and determines if module should be communicated or not
-func (f *Filter) ShouldProcess(path string) bool {
-	rule := f.Rule(path)
-	// process everything unless it's excluded
-	return rule != Exclude
-}
-
 // Rule returns the filter rule to be applied to the given path
 func (f *Filter) Rule(path string) FilterRule {
 	segs := getPathSegments(path)
-	return f.shouldProcess(segs...)
+	rule := f.shouldProcess(segs...)
```

Comment:
`shouldProcess` returns 3 states as well so maybe rename that too?

Reply:
great point @marpio